pkgs/misc/gnash/default.nix

```
@@ -93,6 +93,13 @@ stdenv.mkDerivation {
     ++ optionals enablePlugins   [ xulrunner npapi_sdk ]
     ++ optionals enableGTK       [ gtk2 gnome2.gtkglext gnome2.GConf ];
 
+  patches = [
+    (fetchpatch { # fix compilation due to bad detection of libgif version: https://savannah.gnu.org/patch/index.php?9873
+      url = "https://savannah.gnu.org/patch/download.php?file_id=47859";
```

I got a redirect to this URL, may help with the hash mismatch
```suggestion
      url = "https://savannah.gnu.org/file/DGifCloseFile-update.patch?file_id=47859";
```

---

Redirects do not affect hash mismatches (unless they change to point to a different file). The only thing that matters for fixed output derivations is the final content. Here, the output after passing the patch through `filterdiff`, so the hash will not match the one of unprocessed patch as described above.